permissions/perUserMfaState/revokePermission.ps1

```diff
@@ -0,0 +1,322 @@
+###############################################
```

Comment:
Please use the default headers and add this as comment. Also not -> note

Reply:
default header toegevoegd